test/viewmodels/DeclarationDetailViewModelSpec.scala

```diff
@@ -37,50 +37,70 @@ class DeclarationDetailViewModelSpec extends SpecBase {
 
   "DeclarationDetailViewModel" should {
 
-    "generate correctly declarationSummaryList data" in new Setup {
+    "generate correctly declarationSummaryList data with normalized HTML content" in new Setup {
 
       val summaryList: _root_.uk.gov.hmrc.govukfrontend.views.Aliases.SummaryList =
         DeclarationDetailViewModel.declarationSummaryList(declaration)
 
-      val extractedResultData: Seq[(String, String)] = summaryList.rows.map(row => (
-        row.key.content.asInstanceOf[HtmlContent].asHtml.toString,
-        row.value.content.asInstanceOf[HtmlContent].asHtml.toString
-      ))
+      val extractedResultData: Seq[(String, String)] = summaryList.rows.map { row =>
+        (
+          normalizeHtml(row.key.content.asInstanceOf[HtmlContent].asHtml.toString),
+          normalizeHtml(row.value.content.asInstanceOf[HtmlContent].asHtml.toString)
+        )
+      }
 
       val expectedDeclarationSummaryData: Seq[(String, String)] = Seq(
-        (messages("cf.cash-account.csv.date"), Formatters.dateAsDayMonthAndYear(declaration.date)),
-        (messages("cf.cash-account.csv.movementReferenceNumber"), declaration.movementReferenceNumber),
-        (messages("cf.cash-account.csv.uniqueConsignmentReference"), declaration.declarantReference
-          .getOrElse(emptyString)),
-        (messages("cf.cash-account.csv.declarantEori"), declaration.declarantEori),
-        (messages("cf.cash-account.csv.importerEori"), declaration.importerEori.getOrElse(emptyString))
+        (normalizeHtml(messages("cf.cash-account.csv.date")), Formatters.dateAsDayMonthAndYear(declaration.date)),
+        (normalizeHtml(
+          s"""
+            ${messages("cf.cash-account.csv.declaration")}
+            <abbr title="${messages("cf.cash-account.detail.movement-reference-number")}">
+              ${messages("cf.cash-account.detail.mrn")}
+            </abbr>
+          """), declaration.movementReferenceNumber),
```

Comment:
might be worth adding some spacing before each normalizeHtml line just to increase readability